packages/manager/apps/pci-kubernetes/src/hooks/useMergedFlavorById.ts

```diff
@@ -0,0 +1,34 @@
+import { useMemo } from 'react';
+import { useMergedKubeFlavors } from './useFlavors';
+import { TComputedKubeFlavor } from '@/components/flavor-selector/FlavorSelector.component';
+
+export const getPriceByDesiredScale = (
+  hour?: number,
+  month?: number,
+  desiredScaling = 0,
+) => {
```

Comment:
Could you add the return type of the function, to make it explicit? (as it's exported)